json-examples/slice-provisionning.json

```diff
@@ -0,0 +1,137 @@
+        "sdps": {
+          "sdp": [
+            {
+              "sdp-id": "sdp1",
+              "ac-ref": [
+                {
+                  "name":"ac1"
+                }
+              ]
+            },
+            {
+              "sdp-id": "sdp2",
+              "ac-ref": [
+                {
+                  "name":"ac2"
+                }
+              ]
+            }
+          ]
+        }
+      }
+    ]
+  },
+  "ietf-ac-svc:attachment-circuits": {
+    "ac": [
+      {
+        "name": "ac1",
+        "description": "Connection to site1 on vlan 100 for slice EMBB_UP",
+        "requested-ac-start": "2023-12-12T05:00:00.00Z",
+        "l2-connection": {
+          "encapsulation": {
+            "type": "ietf-vpn-common:dot1q",
+            "dot1q": {
+              "tag-type": "ietf-vpn-common:c-vlan",
+              "cvlan-id": 100
+            }
+          },
+          "bearer-reference": "bearerX@site1"
+        },
+        "ip-connection": {
+          "ipv4": {
+            "local-address": "172.30.1.2",
+            "prefix-length": 30,
+            "address": [
+              {
+                "address-id": "1",
+                "customer-address": "172.30.1.1"
+              }
+            ]
+          }
+        },
+        "routing-protocols": {
+          "routing-protocol": [
+            {
+              "id": "1",
+              "type": "??????????? makes sense or not ?",
+              "static": {
+                "cascaded-lan-prefixes": {
+                  "ipv4-lan-prefixes": [
+                    {  
+                      "lan": "192.168.100.0/24",
+                      "next-hop": "172.30.1.1",
+                      "lan-tag": "primary_UP_slice"
+                    }      
+                  ]  
+                }    
+              }
+            }
+          ]
+        }
+      },
+      {
+        "name": "ac2",
+        "description": "Connection to site2 on vlan 200 for slice EMBB_UP",
+        "requested-ac-start": "2023-12-12T05:00:00.00Z",
+        "l2-connection": {
+          "encapsulation": {
+            "type": "ietf-vpn-common:dot1q",
+            "dot1q": {
+              "tag-type": "ietf-vpn-common:c-vlan",
+              "cvlan-id": 200
+            }
+          },
+          "bearer-reference": "bearerY@site2"
+        },
+        "ip-connection": {
+          "ipv4": {
+            "local-address": "172.30.2.2",
+            "prefix-length": 30,
+            "address": [
+              {
+                "address-id": "1",
+                "customer-address": "172.30.2.1"
+              }
+            ]
+          }
+        },
+        "routing-protocols": {
+          "routing-protocol": [
+            {
+              "id": "1",
+              "type": "??????????? makes sense or not ?",
+              "bgp": {
+                "neighbor": [
+                  {   
+                      "id": "1",
+                      "remote-address":"172.30.2.1",
```

Comment:
The addresses can be removed. The primary @ in the ip-connection will be used by default.